scripts/license-headers.sh

```diff
@@ -25,6 +25,7 @@ ignores=(
   "-ignore=e2e/testdata/helm-charts/**"
   "-ignore=.output/**"
   "-ignore=e2e/testdata/*.xml"
+  "-ignore=.idea/**"
```

Comment:
Add .vscode while you're at it.
Maybe also to the .dockerignore.
Both are in .gitignore

Reply:
done